<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configwidget</class>
 <widget class="QWidget" name="configwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>303</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>444</width>
    <height>303</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>444</width>
    <height>303</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>11</y>
         <width>401</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labDirectory">
            <property name="text">
             <string>Default save directory:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="editDir">
              <property name="toolTip">
               <string>Path to default selection dir for saving</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonBrowse">
              <property name="toolTip">
               <string>Browse filesystem</string>
              </property>
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="labFilename">
              <property name="text">
               <string>Default filename:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editFileName">
              <property name="toolTip">
               <string>Default filename</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="labFormat">
              <property name="text">
               <string>Format</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbxFormat">
              <property name="toolTip">
               <string>Defaukt saving image format</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Delay:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="defDelay">
            <property name="toolTip">
             <string>Default delya before grabbing screen</string>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string> sec</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <number>90</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkNoDecorX11">
            <property name="text">
             <string>No window decoration</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabAdvanced">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>11</y>
         <width>401</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkIncDate">
          <property name="toolTip">
           <string>Inserting current date time into asved filename</string>
          </property>
          <property name="text">
           <string>Insert DateTime in filename</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="labMask">
            <property name="text">
             <string>Template: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editDateTmeTpl">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>256</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labMaskExample">
          <property name="text">
           <string>Example: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkAutoSave">
          <property name="toolTip">
           <string>Automaticaly saving screenshots in grabbing process</string>
          </property>
          <property name="text">
           <string>Autosave screen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkAllowCopies">
          <property name="toolTip">
           <string>Allow run multiplies copy of Screen Grab</string>
          </property>
          <property name="text">
           <string>Allow multiple copies</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Display</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QCheckBox" name="checkSaveSize">
         <property name="toolTip">
          <string>Saving main window size on exit</string>
         </property>
         <property name="text">
          <string>Save window size on exit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkZommMouseArea">
         <property name="text">
          <string>Zoom area around mouse in selection mode</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Tray</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>401</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkShowTray">
          <property name="text">
           <string>Show tray</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="labTrayMessages">
            <property name="text">
             <string>Tray messages:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbxTrayMsg">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Tray messages display mode</string>
            </property>
            <item>
             <property name="text">
              <string>Never</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tray mode</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Always</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="labTimeTrayMess">
            <property name="text">
             <string>Time of display tray messages</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="timeTrayMess">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Time to display tray messages</string>
            </property>
            <property name="suffix">
             <string> sec</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkInTray">
          <property name="toolTip">
           <string>Minimize in tray on click close button</string>
          </property>
          <property name="text">
           <string>Close in tray</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Hotkeys</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QTreeWidget" name="treeKeys">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <column>
          <property name="text">
           <string>Action</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Hotkey</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Glpbal hotkeys</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <item>
           <property name="text">
            <string>Fill screen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Active window</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Area select</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Local hotkeys</string>
          </property>
          <item>
           <property name="text">
            <string>New screen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Save screen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Copy screen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Options</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Help</string>
           </property>
          </item>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="labUsedShortcut">
           <property name="text">
            <string>Used shortcut:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QKeySequenceWidget" name="keyWidget">
           <property name="noneText">
            <string>Not defined</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="butRestoreOpt">
       <property name="toolTip">
        <string>restore default settings</string>
       </property>
       <property name="text">
        <string>Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>88</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="butSaveOpt">
       <property name="toolTip">
        <string>Save settings</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butCancel">
       <property name="toolTip">
        <string>Discard changes</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QKeySequenceWidget</class>
   <extends></extends>
   <header>qkeysequencewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>editDir</tabstop>
  <tabstop>buttonBrowse</tabstop>
  <tabstop>editFileName</tabstop>
  <tabstop>cbxFormat</tabstop>
  <tabstop>defDelay</tabstop>
  <tabstop>butRestoreOpt</tabstop>
  <tabstop>butSaveOpt</tabstop>
  <tabstop>butCancel</tabstop>
  <tabstop>checkIncDate</tabstop>
  <tabstop>checkAutoSave</tabstop>
  <tabstop>checkSaveSize</tabstop>
  <tabstop>checkInTray</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
